### ISSUE TITLE
Generic.Arrays.ArrayIndent is indenting sub-arrays incorrectly when comma not used after the last value

### ISSUE THREAD

**Issue by michaelarnauts** [squizlabs/PHP_CodeSniffer#1881]
Generic.Arrays.ArrayIndentSniff is indenting the first line of the last entry incorrectly.

```php
<?php

return [
    [
        'id' => 1,
        'status' => 2,
    ],
    [
        'id' => 2,
        'status' => 2,
    ],
    [
        'id' => 3, // <- line 13
        'status' => 2,
    ] // <- no comma
];
```
Throws:
```
 13 | ERROR | [x] Array key not indented correctly; expected 4 spaces but found 8 (Generic.Arrays.ArrayIndent.KeyIncorrect)
```

When indenting it with 4 spaces:
```
 13 | ERROR | [x] Array key not indented correctly; expected 8 spaces but found 4 (Generic.Arrays.ArrayIndent.KeyIncorrect)
```

And now, for fun, when indenting it with 6 spaces.
```
 13 | ERROR | [x] Array key not indented correctly; expected 4 spaces but found 6 (Generic.Arrays.ArrayIndent.KeyIncorrect)
 13 | ERROR | [x] Array key not indented correctly; expected 8 spaces but found 6 (Generic.Arrays.ArrayIndent.KeyIncorrect)
```

I did notice that when I add an comma to the last item (marked with `// <- no comma`), the Sniff works fine.

**Comment by gsherwood** [comment]
Thanks for reporting this. It will be fixed in 3.3.0
